AreaShop/src/main/java/me/wiefferink/areashop/commands/CommandAreaShop.java

```diff
@@ -81,11 +86,44 @@ public boolean confirm(CommandSender sender, String[] args, Message message) {
 			return true;
 		}
 
-		message.prefix().append(Message.fromKey("confirm-yes").replacements(command)).send(sender);
+		Message m = message.prefix().append(Message.fromKey("confirm-yes").replacements(command));
+		send(m, sender);
 		lastUsed.put(sender.getName(), new CommandTime(command, now));
 		return false;
 	}
 
+	private void send(Message message, CommandSender target) {
```

Comment:
I see this method is basically copy-pasted across multiple command files. Refactor this to a utility method.